sdk/eventhub/event-hubs/package.json

```diff
@@ -35,20 +35,16 @@
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:test": "npm run build",
```

Comment:
Just this change should be enough to avoid building browser.

Reply:
I'm getting the same error only with ` "build:test": "npm run build",` change

Reply:
Reverted extra changes!!